### PR TITLE
update android release-notes for v2.4.0 

### PR DESCRIPTION
**Pull request by JohnPeters7116**
## Description

Please describe what the changes are made in this pull request and why the change was necessary.

Update release notes for v2.4.0

## Deploy Notes

Notes regarding deployment the contained body of work. These should note any db migrations, etc.

### CODE REVIEW

File: _documentation/en/client-sdk/sdk-documentation/android/release-notes.md [hunk 1]
```diff
@@ -6,6 +6,43 @@ navigation_weight: 0
 
 # Release Notes
 
+## Version 2.4.0 - March 3 , 2020
+
+### Added
+
+- Add filter by state to `getConversationsPage`  in `NexmoClient`.
+
+```NexmoClient.get().getConversationsPage(50, NexmoPageOrderDesc, "JOINED", new NexmoRequestListener<NexmoConversationsPage>(){
```
Comment (tbedford) [inline _documentation/en/client-sdk/sdk-documentation/android/release-notes.md]: You need a new line after the triple back ticks.

File: _documentation/en/client-sdk/sdk-documentation/android/release-notes.md [hunk 2]
```diff
@@ -6,6 +6,43 @@ navigation_weight: 0
...
+        void onError(@NonNull NexmoApiError error){
+
+           }
+           void onSuccess(@Nullable NexmoConversationsPage result){
+
+           }
+        })
+   }
+});
+
```
Comment (tbedford) [inline _documentation/en/client-sdk/sdk-documentation/android/release-notes.md]: You can remove the blank line before the triple back ticks.

File: _documentation/en/client-sdk/sdk-documentation/android/release-notes.md (hunk 1 above)
Comment (JohnPeters7116) [reply _documentation/en/client-sdk/sdk-documentation/android/release-notes.md]: done

File: _documentation/en/client-sdk/sdk-documentation/android/release-notes.md (hunk 2 above)
Comment (JohnPeters7116) [reply _documentation/en/client-sdk/sdk-documentation/android/release-notes.md]: done

**Pull request merged**